tests/core/numerics/pusher/test_pusher.cpp

```diff
@@ -221,20 +226,22 @@ TEST_F(APusher2D, trajectoryIsOk)
 
 TEST_F(APusher1D, trajectoryIsOk)
 {
-    auto rangeIn  = makeRange(std::begin(particlesIn), std::end(particlesIn));
-    auto rangeOut = makeRange(std::begin(particlesOut), std::end(particlesOut));
+    auto rangeIn  = makeIndexRange(particlesIn);
+    auto rangeOut = makeIndexRange(particlesOut);
     std::copy(rangeIn.begin(), rangeIn.end(), rangeOut.begin());
 
     for (decltype(nt) i = 0; i < nt; ++i)
     {
         actual[0][i]
             = (particlesOut[0].iCell[0] + particlesOut[0].delta[0]) * static_cast<float>(dxyz[0]);
 
-        pusher->move(rangeIn, rangeOut, em, mass, interpolator, selector, layout);
+        pusher->move(rangeIn, rangeOut, em, mass, interpolator, layout, selector, selector);
 
         std::copy(rangeOut.begin(), rangeOut.end(), rangeIn.begin());
     }
 
+    std::cout << actual[0][985] << " " << expectedTrajectory.x[985] << " " << actual[0][986] << " "
```

Comment:
?

Reply:
you don't like these particles 985 and 986?
removed:)